### DISCUSSION TITLE
Make Temporary Override feature easily configurable by first-time users

### DISCUSSION THREAD

**Discussion by Anton-V-K** [proginosko/LeechBlockNG#382] [Ideas]
LeechBlock Override just asks for the duration, disabling the blocking completely WITHOUT requesting the password, which was set to access options.

**Comment by proginosko** [comment]
I just tested it and it seems to work fine. Did you select "Require the user to enter the access control password (see above)" in the Temporary Override section of General options?

**Comment by Anton-V-K** [comment]
Thanks for the reply. Indeed I've missed this option in the Temporary Override section. The feature now works as needed :)
I'd expect, it would be set automagically if Temporary Override is enabled and global password is set, otherwise blocking rules are easily skipped.
Or the option title "Allow temporary override for these sites" may be adjusted in the accordance with the mode.
Though for an easy fix it would be enough to add a hint near the option "Allow temporary override for these sites" to remind the user about checking the General page.
